src/dotnet/APIView/APIViewUITests/SmokeTests.cs

```diff
@@ -11,7 +11,7 @@ namespace APIViewUITests
 {
     public class SmokeTests
     {
-        [Fact]
+        [Fact(Skip = "Reason")]
```

Comment:
could we add more descriptive message on why these are flaky?

Reply:
Had that but failed to save the file before committing.